spec/controllers/account_reset/recover_controller_spec.rb

```diff
@@ -0,0 +1,109 @@
+require 'rails_helper'
+
+describe AccountReset::RecoverController do
+  let(:user) { create(:user, :with_authentication_app, :with_email) }
+  before do
+    allow_any_instance_of(UserDecorator).to receive(:identity_verified?).and_return(true)
```

Comment:
You would not have to use `allow_any_instance_of` if you created your user with a profile a couple lines up